tests/test_fusor.py

```diff
@@ -491,6 +494,44 @@ def fusion():
     }
 
 
+@pytest.fixture(scope="module")
+def fusion_custom_sequence_id():
+    """Create fixture using custom/unrecognized sequence ID"""
+    params = {
+        "structural_components": [
+            {
+                "component_type": "templated_sequence",
+                "strand": "+",
+                "region": {
+                    "id": "fusor.location_descriptor:NM_152263.3",
+                    "type": "LocationDescriptor",
+                    "location": {
+                        "sequence_id": "refseq:chr12",
```

Comment:
I'm unsure if this is standard notation (assuming we want to go with possible real examples). When using a chromosome with a prefix, I've only seen `assembly:chr`. With refseq prefixes, I've only seen `NC_` accessions

Reply:
I was trying to go with something fake/custom/unrecognizable (to make sure it doesn't break if you give it like a bespoke or in-house accession ID). I went back and added some cases for Ensembl IDs (which HGVS can get the namespace for, but SeqRepo can't translate I think?) and then a more obviously fake ID